src/zcl_abapgit_repo.clas.abap

```diff
@@ -96,6 +95,14 @@ CLASS zcl_abapgit_repo DEFINITION
         !is_settings TYPE zif_abapgit_persistence=>ty_repo-local_settings
       RAISING
         zcx_abapgit_exception .
+    METHODS get_local_checksums
+      RETURNING
+        VALUE(rt_checksums) TYPE zif_abapgit_persistence=>ty_local_checksum_tt .
+    METHODS set_local_checksums
```

Comment:
can we wait moving scope / introducing set_local_checksums until it is needed? the repo class is too big already, been doing a lot of refactoring to have less public methods